.ci/scripts/install-tools.bat

```diff
@@ -7,9 +7,19 @@ IF ERRORLEVEL 1 (
  choco install curl -y --no-progress --skipdownloadcache
 )
 mkdir %WORKSPACE%\bin
+
+REM If 32 bits then install the GVM accordingly
+IF NOT EXIST "%PROGRAMFILES(X86)%" (
+    curl -sL -o %WORKSPACE%\bin\gvm.exe https://github.com/andrewkroh/gvm/releases/download/v0.2.2/gvm-windows-386.exe
+)
+
```

Comment:
NOTE: this could be removed as soon as the packer images have been deployed with the 32 bit support